app/javascript/react/components/MonsterTile.js

```diff
@@ -4,7 +4,7 @@ const MonsterTile = props => {
   const { monster } = props
 
   return (
-    <div className="callout secondary cell small-6 row grid-x">
+    <div className="callout secondary cell small-6 row grid-x" id="monster-tile">
```

Comment:
`id` selectors are meant to be unique, the webpage should only have one element with a given `id` property, and it looks like you're giving this `id` to all of your Monster Tiles.  A better approach would be to add `monster-tile` as a class name and use the `class` for applying your style